.github/ISSUE_TEMPLATE/Other.md

```diff
@@ -1,10 +0,0 @@
----
-name: Other
-about: Something else
----
-
-<!-- Bug reports and Feature requests must use other templates, or will be closed -->
-<!-- Please ask questions on StackOverflow or the GitHub Discussions. -->
-<!-- https://github.com/webpack/webpack/discussions -->
-<!-- https://stackoverflow.com/questions/ask?tags=webpack -->
-<!-- Issues which contain questions or support requests will be closed. -->
```

Comment:
@3ru We want to bring this back, sometimes some messages can't be attributed to bugs or features, but it could potentially be, what we need here - textarea + environment

Reply:
@3ru Even let's do it better union idea + other and return it is a issue template

discussions are in free form, it's just questions and answers, I don't think it's worth forcing people to waste time filling in fields in such a context

Reply:
Just experience, sometimes a developer just can't formalize a feature request properly, or they don't understand that it's a bug/problem, yes we can always move the discussion to the issues, I just prefer that there will always be an alternative simple form for such purposes